components/theme-switcher.tsx

```diff
@@ -1,22 +1,36 @@
 import { useAtom } from 'jotai'
-import { Select } from 'react-daisyui'
+import { Dropdown, Button } from 'react-daisyui'
+import { Theme } from '@icon-park/react'
+
 import { persistentThemeAtom } from '../src/atoms'
 
+const DEFAULT_THEMES = ['light', 'dark', 'auto']
+
+function capitalize(str: string) {
+  return str.charAt(0).toUpperCase() + str.slice(1)
+}
+
 export default function ThemeSwitcher() {
   const [persistentTheme, setPersistentTheme] = useAtom(persistentThemeAtom)
+  const handleChangeThemeGen = (theme: 'light' | 'dark' | 'auto') => () => {
```

Comment:
this kind of *Gen function seems like meaningless.

Reply:
Fixed.